### DISCUSSION TITLE
How many Grand Slam singles titles has Serena Williams won?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#88] [Galaxy Trivia] [answered]
**Hint:** 🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆🏆

**Comment by Tynab** [comment, marked as answer]
Serena Williams has won 23 Grand Slam singles titles in her tennis career, which is the most by any player in the Open Era of tennis. She won her first Grand Slam singles title at the 1999 US Open, and her most recent Grand Slam singles title was the 2017 Australian Open. Her Grand Slam singles titles include 7 Australian Open titles, 3 French Open titles, 7 Wimbledon titles, and 6 US Open titles. Serena Williams is widely regarded as one of the greatest tennis players of all time, and her achievements in the sport are truly remarkable.
